tools/memap.py

```diff
@@ -62,23 +63,69 @@ def __init__(self, detailed_misc=False):
 
         self.subtotal = dict()
 
-    def module_add(self, module_name, size, section):
-        """ Adds a module / section to the list
+        self.misc_flash_mem = 0
+
+
+    def module_remove_unused(self):
```

Comment:
Could we rename this `remove_unused_modules`? That name better matches what this function does.